source/common/formatter/stream_info_formatter.cc

```diff
@@ -1352,6 +1353,54 @@ const StreamInfoFormatterProviderLookupTable& getKnownStreamInfoFormatterProvide
                     return result;
                   });
             }}},
+          {"UPSTREAM_PEER_URI_SAN",
+           {CommandSyntaxChecker::COMMAND_ONLY,
+            [](const std::string&, absl::optional<size_t>) {
+              return std::make_unique<StreamInfoUpstreamSslConnectionInfoFormatterProvider>(
+                  [](const Ssl::ConnectionInfo& connection_info) {
+                    return absl::StrJoin(connection_info.uriSanPeerCertificate(), ",");
```

Comment:
Why is the comma needed?
One would expect the operator to just return the value, no?

Reply:
There may be multiple entries for a given type of SAN (see the [screenshot](https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Subject_Alt_Names_on_Firefox_90_screenshot.png/600px-Subject_Alt_Names_on_Firefox_90_screenshot.png) here), so we need to delimit each one with a comma

Reply:
My mistake, read it as `StrCat` instead of `StrJoin`.